test/cpp/interop/grpclb_fallback_test.cc

```diff
@@ -160,7 +160,7 @@ std::unique_ptr<TestService::Stub> CreateFallbackTestStub() {
   grpc::ChannelArguments channel_args;
   grpc_socket_mutator* tcp_user_timeout_mutator =
       static_cast<grpc_socket_mutator*>(
-          gpr_malloc(sizeof(tcp_user_timeout_mutator)));
+          gpr_malloc(sizeof(*tcp_user_timeout_mutator)));
```

Comment:
let's change this to use new and delete rather than malloc/free:

grpc_socket_mutator* tcp_user_timeout_mutator = new grpc_socket_mutator();

and then in vtable destroy method: `delete mutator`

Reply:
Done